src/test/java/org/cactoos/map/NoNullsTest.java

```diff
@@ -347,23 +349,21 @@ public void clear() {
     @Test
     @SuppressWarnings("unchecked")
     public void entrySet() {
-        MatcherAssert.assertThat(
-            "Can't call #entrySet()",
+        new Assertion<Set<Map.Entry<Integer, Integer>>>(
+            "Must call #entrySet()",
             new NoNulls<Integer, Integer>(
                 new HashMap<Integer, Integer>() {
                     {
                         put(1, 1);
                         put(0, 0);
                     }
                 }
-            ),
+            ).entrySet(),
             new AllOf<>(
-                new IterableOf<>(
-                    new IsMapContaining<>(new IsEqual<>(1), new IsEqual<>(1)),
-                    new IsMapContaining<>(new IsEqual<>(0), new IsEqual<>(0))
-                )
+                new IsIterableContaining<>(new IsEntry<>(1, 1)),
+                new IsIterableContaining<>(new IsEntry<>(0, 0))
```

Comment:
@andreoss it is because this tests is here to explicitly test the behaviour of a `Map` implementation and so we test the `entrySet` method itself instead of having an implicit expectation that it is what `HasEntry` actually does (which it does not ^^)